<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Point</class>
 <widget class="QDialog" name="Point">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Атрибуты точки</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/plugins/openland/icons/point.png</normaloff>:/plugins/openland/icons/point.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QSpinBox" name="spinBoxNumber">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>Отмена</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/plugins/openland/icons/close.png</normaloff>:/plugins/openland/icons/close.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSpinBox" name="spinBoxOrder">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximum">
      <number>999999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Порядок&lt;br/&gt;обхода&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditMeasureError">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Координата Y</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditY">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="comboBoxMethod">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QComboBox" name="comboBoxWay2Fix">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Префикс&lt;br/&gt;номера&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBoxUpdateGeom">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Обновить геометрию</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Способ&lt;br/&gt;закрепления&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditX">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Метод&lt;br/&gt;определения&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Координата X</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditPrefix">
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="../resources.qrc">
       <normaloff>:/plugins/openland/icons/save.png</normaloff>:/plugins/openland/icons/save.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Погрешность</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Номер&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBoxIsPre">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string>Прекратившая существование</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Уточненная&lt;br/&gt;точка&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QComboBox" name="comboBoxCorrectPoint">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditPrefix</tabstop>
  <tabstop>spinBoxNumber</tabstop>
  <tabstop>lineEditX</tabstop>
  <tabstop>spinBoxOrder</tabstop>
  <tabstop>lineEditY</tabstop>
  <tabstop>comboBoxMethod</tabstop>
  <tabstop>lineEditMeasureError</tabstop>
  <tabstop>comboBoxWay2Fix</tabstop>
  <tabstop>checkBoxIsPre</tabstop>
  <tabstop>comboBoxCorrectPoint</tabstop>
  <tabstop>checkBoxUpdateGeom</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
